Add support for using external binaries that are in the PATH
Author: Samuel Moors (VUB)
diff -ur ete3-3.1.2.orig/ete3/evol/evoltree.py ete3-3.1.2/ete3/evol/evoltree.py
--- ete3-3.1.2.orig/ete3/evol/evoltree.py	2020-08-30 09:08:26.000000000 +0200
+++ ete3-3.1.2/ete3/evol/evoltree.py	2022-05-30 17:38:35.529290893 +0200
@@ -248,6 +248,8 @@
         hlddir = os.getcwd()
         os.chdir(fullpath)
         bin_ = os.path.join(self.execpath, model_obj.properties['exec'])
+        if not os.path.exists(bin_):
+            bin_ = which(model_obj.properties['exec'])
         try:
             proc = Popen([bin_, 'tmp.ctl'], stdout=PIPE, stdin=PIPE)
         except OSError:
